frontend/packages/console-dynamic-plugin-sdk/src/webpack/ConsoleRemotePlugin.ts

```diff
@@ -62,8 +60,11 @@ export class ConsoleRemotePlugin {
         filename: remoteEntryFile,
         exposes: this.pkg.consolePlugin.exposedModules || {},
         overridables: sharedPluginModules.filter((m) => {
-          // Exclude modules not present in webpack managed paths (e.g. node_modules)
-          return Array.from(compiler.managedPaths).some((p) => fs.existsSync(path.resolve(p, m)));
```

Comment:
The root cause was that for some reason, `compiler.managedPaths` was different between development builds
```ts
Set(['node_modules'])
```
vs. production builds
```ts
Set([])
```

This caused `overridables` to be empty for production builds, which consequently broke the `require('@console/xx')` calls at runtime for such builds.